recipes/lfric_reader/build.sh

```diff
@@ -0,0 +1,16 @@
+cd src/cxx
+cmake CMakeLists.txt \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DBUILD_SHARED_LIBS=ON \
+      -DBUILD_TESTING=OFF
```

Comment:
We have some common CMake arguments, please use them.
```suggestion
cmake ${CMAKE_ARGS} CMakeLists.txt \
      -DCMAKE_BUILD_TYPE=Release \
      -DBUILD_SHARED_LIBS=ON \
      -DBUILD_TESTING=OFF
```

Reply:
Implemented in commit below, thanks